src/notebooks_service.py

```diff
@@ -95,10 +97,29 @@ def health():
 @authenticated
 def whoami(user):
     """Return information about the authenticated user."""
-    return Response(
-        json.dumps(user, indent=1, sort_keys=True),
-        mimetype='application/json',
+    headers = {auth.auth_header_name: 'token {0}'.format(auth.api_token)}
+    info = json.loads(
+        requests.request(
+            'GET',
+            '{prefix}/users/{user[name]}'.format(
+                prefix=auth.api_url, user=user
+            ),
+            headers=headers
+        ).text
     )
+    return jsonify(info)
+
+
+@authenticated
```

Comment:
why do have to be the static files accessible only for authenticated users?

Reply:
Because the static files access API endpoints that require authentication. The UI code is simpler if it can assume that the user is already authenticated.

Reply:
But what if we want to serve the files directly using nginx?

Reply:
My understanding of the requirements for #4 is that this solution should be minimal and simple. For Renku, we will create a more full-featured UI. That UI is already being served from nginx.